client/js/store/core/initStore.js

```diff
@@ -72,16 +72,22 @@ function initStore (storeModules, apiStoreModules, presetStoreModules) {
             },
             successCallbacks: [
               async (success) => {
-                const response = await success.json()
+                // If the response body is empty, contentType will be null
+                const contentType = success.headers.get('Content-Type')
 
-                const meta = response.data?.meta
-                  ? response.data.meta
-                  : response.meta || null
-                if (meta?.messages) {
-                  handleResponseMessages(meta)
-                }
+                if (contentType && contentType.includes('application/json')) {
+                  const response = await success.json()
 
-                return Promise.resolve(response)
+                  const meta = response.data?.meta
+                    ? response.data.meta
+                    : response.meta || null
+                  if (meta?.messages) {
+                    handleResponseMessages(meta)
+                  }
+
+                  return Promise.resolve(response)
+                }
+                return Promise.resolve(success)
```

Comment:
```suggestion
                }

                return Promise.resolve(success)
```

Reply:
Done in d1470c717c28094fabe8fcc0b6e7fa0cc47f3e82